codersdk/deployment.go

```diff
@@ -128,6 +128,17 @@ func (n FeatureName) AlwaysEnable() bool {
 	}[n]
 }
 
+// Premium returns true if the feature is a premium feature.
+func (n FeatureName) Premium() bool {
+	switch n {
+	// Add all features that should be excluded in the Enterprise feature set.
+	case FeatureMultipleOrganizations, FeatureCustomRoles:
+		return true
+	default:
+		return false
+	}
+}
```

Comment:
This is a bit confusing since all enterprise features are also enterprise features. Maybe instead do `Enterprise()` and flip the boolean? Premium is "Everything"

Reply:
Good point, will flip